x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -341,5 +341,57 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         });
       });
     });
+
+    describe('#With Asset Details using nginx host', () => {
+      before(async () => {
+        await Promise.all([
+          esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+          kibanaServer.savedObjects.cleanStandardList(),
+        ]);
+        await browser.setWindowSize(1600, 1200);
+
+        await navigateToNodeDetails('demo-stack-nginx-01', 'demo-stack-nginx-01');
+        await pageObjects.header.waitUntilLoadingHasFinished();
+      });
+
+      after(async () => {
+        await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
+      });
+
+      describe('Overview Tab Nginx', () => {
+        before(async () => {
+          await pageObjects.assetDetails.clickOverviewTab();
+
+          await pageObjects.timePicker.setAbsoluteRange(
+            START_HOST_ALERTS_DATE.format(DATE_PICKER_FORMAT),
+            END_HOST_ALERTS_DATE.format(DATE_PICKER_FORMAT)
+          );
+        });
+
+        [
+          { metric: 'cpuUsage', value: '0.8%' },
+          { metric: 'normalizedLoad1m', value: '1.4%' },
+          { metric: 'memoryUsage', value: '18.0%' },
+          { metric: 'diskSpaceUsage', value: '17.5%' },
+        ].forEach(({ metric, value }) => {
+          it(`${metric} tile should show ${value}`, async () => {
+            await retry.tryForTime(3 * 1000, async () => {
+              const tileValue = await pageObjects.assetDetails.getAssetDetailsKPITileValue(metric);
+              expect(tileValue).to.eql(value);
+            });
+          });
+        });
+
+        it('should render 12 charts in the Metrics section', async () => {
+          const hosts = await pageObjects.assetDetails.getAssetDetailsMetricsCharts();
+          expect(hosts.length).to.equal(12);
+        });
+
+        it('should render 3 charts in the Nginx Metrics section', async () => {
+          const hosts = await pageObjects.assetDetails.getAssetDetailsNginxMetricsCharts();
+          expect(hosts.length).to.equal(3);
```

Comment:
Q: Is there any possibility to validate the existence of the 4th, Active Connections, chart?

Reply:
It is but there is a tricky part: We need to change the existing fixture to have also filebeat index and set `filebeat-*` in the setting before we run the test. I am not sure if that would affect the existing test results. 
Another probably better option is to use the new host snapshot we use in the details view and replace it or add filebeat data only there so it won't affect another test (we can still check first if we can use filebeat index to create the snapshot or just get a local document and replace dates/hostname to match the existing test data) and set `filebeat-*` in the setting before we run the test as well in this option.

Reply:
Yeah. Sounds complicated. I think it will be easier if we use the logs data view for this chart. We can get back to this when we start investigating https://github.com/elastic/kibana/issues/165818